dependency.lic

```diff
@@ -770,8 +770,20 @@ class SetupFiles
     boxes = @base_files['base-items.yaml'][:box_nouns]
     gems = @base_files['base-items.yaml'][:gem_nouns]
     scrolls = @base_files['base-items.yaml'][:scroll_nouns]
-    s.lootables = (base_lootables + boxes + gems + scrolls + s.loot_additions - s.loot_subtractions).uniq
+    spells = @base_files['base-spells.yaml'][:spell_data]
 
+    s.lootables = (base_lootables + boxes + gems + scrolls + s.loot_additions - s.loot_subtractions).uniq
+    s.offensive_spells
```

Comment:
It looks like there are waggle, lockpicking, astrology and possibly thievery and athletics? I'm not familiar with the latter two. I kinda think these things all belong under `waggle_sets` but that may be a bit of work (im up for it though).